src/pages/index.js

```diff
@@ -113,60 +178,72 @@ function openEditForm() {
   editFormPopup.open();
 }
 
-function submitEditProfile(inputValues) {
-  editFormPopup.renderLoading(true);
-  return api
-    .editUserInfo(inputValues.title, inputValues.description)
-    .then(() => {
-      userInfo.setUserInfo(inputValues.title, inputValues.description);
-      editFormPopup.close();
-    })
-    .catch((err) => {
-      console.log(err);
-    })
-    .finally(() => {
-      editFormPopup.renderLoading(false, "Save");
-    });
-}
-
-api
-  .getApiInfo()
-  .then(([userData, userCards]) => {
-    userId = userData._id;
-    userInfo.setUserInfo(userData);
-    userInfo.setAvatar(userData);
-    const cardSection = new Section(
-      { items: userCards, renderer: renderCard },
-      containerSelectors.cardSection
-    );
-    cardSection.renderItems();
-  })
-  .catch((err) => {
-    console.log(err);
-  });
+/* -------------------------------- add form -------------------------------- */
 
 function openAddForm() {
-  addFormPopup.open();
+  addCardPopup.open();
   addFormValidator.toggleButtonState();
 }
 
-function submitAddCard(data) {
-  renderCard({ name: data.title, link: data.link });
-}
+/* ------------------------------ avatar popup ------------------------------ */
 
-function renderCard(data) {
-  const card = createCard(data);
-  cardSection.addItem(card);
+function openAvatarForm() {
+  avatarPopup.open();
+  avatarFormValidator.toggleButtonState();
 }
 
-function createCard(item) {
-  const cardElement = new Card(
-    item,
-    containerSelectors.cardTemplate,
-    handleImageClick
-  ).getView();
-  return cardElement;
-}
+/* ---------------------------------- cards --------------------------------- */
+
+const createCard = (cardData) => {
+  const cardElement = new Card({
+    data: cardData,
+    userId: userId,
+    cardSelector: containerSelectors.cardTemplate,
+    handleImageClick,
+    handleDeleteClick: (cardId) => {
+      deleteCardPopup.open();
+      deleteCardPopup.setSubmit(() => {
+        deleteCardPopup.renderLoading(true);
+        api
+          .deleteUserCard(cardId)
+          .then(() => {
+            cardElement.handleDeleteCard();
+            deleteCardPopup.close();
+          })
+          .catch((err) => {
+            console.log(err);
+          })
+          .finally(() => {
+            deleteCardPopup.renderLoading(false, "Save");
+          });
+      });
+    },
+    handleLikeClick: (cardId) => {
+      if (cardElement.isLiked()) {
+        api
+          .deleteLikes(cardId)
+          .then((res) => {
+            cardElement.setLikesCounter(res.likes);
+          })
+          .catch((err) => {
+            console.log(err);
+          });
+      } else {
+        api
+          .addLikes(cardId)
+          .then((res) => {
+            cardElement.setLikesCounter(res.likes);
+          })
+          .catch((err) => {
```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors